doc/mgba-qt.6

```diff
@@ -0,0 +1,97 @@
+.\" Copyright (c) 2015 Anthony J. Bentley <anthony@anjbe.name>
+.\"
+.\" This Source Code Form is subject to the terms of the Mozilla Public
+.\" License, v. 2.0. If a copy of the MPL was not distributed with this
+.\" file, you can obtain one at https://mozilla.org/MPL/2.0/.
+.Dd July 29, 2015
+.Dt MGBA-QT 6
+.Os
+.Sh NAME
+.Nm mgba-qt
+.Nd Game Boy Advance emulator
+.Sh SYNOPSIS
+.Nm mgba-qt
+.Op Fl b Ar biosfile
+.Op Fl l Ar loglevel
+.Op Fl p Ar patchfile
+.Op Fl s Ar n
+.Ar file
+.Sh DESCRIPTION
+.Nm
+is a Game Boy Advance emulator.
+The options are as follows:
+.Bl -tag -width Ds
+.It Fl b Ar biosfile , Fl -bios Ar biosfile
+Specify a BIOS file to use during boot.
+If this flag is omitted,
+.Nm
+will use a high\(hylevel emulated BIOS.
```

Comment:
The default depends on the stored config. What's described is the default if no stored config is present.